evo_researcher/autonolas/research.py

```diff
@@ -1151,5 +1155,24 @@ def make_prediction(prompt: str, additional_information: str, **kwargs) -> Predi
         "additional_information": additional_information,
         "timestamp": formatted_time_utc,
     })
+    response = json.loads(clean_completion_json(response))
```

Comment:
I moved `clean_completion_json` to this file and used it here, because it's typed to return `Prediction` dict, not string:

```
def make_prediction(prompt: str, additional_information: str, **kwargs) -> Prediction
```